docs/compiler/Introduction.md

```diff
@@ -0,0 +1,57 @@
+<!-- README Document on how to build and use this Neural Network Compiler -->
+
+# Introduction
+
+This document discusses the ONE compiler. ONE compiler analyses, quantizes and optimizes the input model. It takes many model formats and converts to a common [Circle](#circle-format)  format which is then deployed.
+
+<!-- Contents -->
+# Contents 
+* [Build Setup](#Build-Setup)
+* [Supported Formats](#Supported-Formats)
+* [Circle Format](#Circle-Format)
+
+<!-- Build Setup -->
+## Build Setup
+<!-- System Requirments -->
+* System Setup   
+This setup is to build the ONE compiler on Ubuntu. Its tested on Ubuntu 16.04 as well as Ubuntu 18.04.  
+Get the code [Samsung ONE](https://github.com/Samsung/ONE) from Github. 
+
+<!-- Software/Package Requirements -->
+* Dependencies
+    * Cmake (Greater than or equal to 3.1)
+    * g++ toolchain (Greater than or equal to 6)
+    * python virtual environment (python3-venv for python3)
+    * HD5 library (libhdf5-dev)
+
+<!-- Build Configuration -->
+* Configure  
+./nncc configure   
```

Comment:
I think this line would be better with

   ```
   ./nncc configure   
   ```

Reply:
Agreed, it would look better in code block.